draft-ietf-capport-api.md

```diff
@@ -75,6 +78,17 @@ For example, if the Captive Portal API server is hosted at example.org, the URI'
 
   - "https://example.org/captive-portal/api"
   - "https://example.org/captive-portal/api/X54PD"
+  
+### Server Authentication
+
+The purpose of accessing the Captive Portal API over an HTTPS connection is twofold: first, the encrypted connection protects the integrity and confidentiality of the API exhange from other parties on the local network; and second, it provides the client of the API an opportunity to authenticate the server that is hosting the API. This authentication is aimed at allowing a user to be reasonably confident that the entity providing the Captive Portal API has a valid certificate for the hostname in the URI (such as "example.com"). The hostname of the API SHOULD be displayed to the user in order to indicate the entity which is providing the API service.
+
+The client SHOULD validate the status of the certificate presented by the API server using OCSP {{RFC6960}}. However, communicating with an OCSP server may be a challenge on a captive network, since general network access is blocked. There are two options the API server can employ to solve this problem:
+
+1. API servers SHOULD support OCSP stapling via the TLS Certificate Status Request extension {{RFC6066}}. This avoids the client directly communicating with an OCSP server.
+2. Captive networks SHOULD allow (whitelist) traffic from the client to OCSP servers in order to allow status validation. This whitelist may be limited to lightweigth OSCP queries {{RFC5019}}.
```

Comment:
I don't know how much 5019 is implemented in clients.  Limiting the whitelist in this way might cause this to not work.  (typo on lightweight, btw)